aiocouch/remote.py

```diff
@@ -246,7 +246,7 @@ async def _get(self) -> JsonDict:
     @raises(401, "CouchDB Server Administrator privileges required")
     @raises(412, "Database already exists")
     async def _put(self, **params: Any) -> JsonDict:
-        _, json = await self._remote._put(self.endpoint)
+        _, json = await self._remote._put(self.endpoint, **params)
```

Comment:
The [`_put` method](https://github.com/metricq/aiocouch/blob/master/aiocouch/remote.py#L109) has `data` as the second argument and `params` shouldn't be expanded. I think it should simply look like this:

```suggestion
        _, json = await self._remote._put(self.endpoint, params=params)
```

Reply:
Sorry, I overlooked that the kwargs are swallowed again in that method, I saw that they were passed along to _request.

What would you think about adding a kwargs parameter there and allow for the same method signature as aiohttp by passing kwargs through from the first call into aiocouch until they reach the couchdb?

Reply:
Also: I saw that some tests started to fail because the parameters are no longer ignored. Maybe I need to revisit this whole pull request.